### DISCUSSION TITLE
sling database

### DISCUSSION THREAD

**Discussion by trung** [slingdata-io/sling-cli#364] [General]
I noticed that when running sling, it creates a sqlite database for internal use.

Just wondering what it is used for?

My use case is that if I run Sling in a docker container without mounting the `SLING_HOME_DIR` and only passing the connections and replication yaml, the sqlite DB will be recreated every run. Would that impact the `incremental` replication mode?

**Comment by flarco** [comment]
It keeps some local history and internal settings. Does not impact runs.
